sources/packages/backend/apps/api/src/testHelpers/auth/token-helpers.ts

```diff
@@ -0,0 +1,111 @@
+import { JwtService } from "@nestjs/jwt";
+import { UserPasswordCredential } from "@sims/utilities/config";
+import { AuthorizedParties, KeycloakConfig } from "../../auth";
+import { KeycloakService, TOKEN_RENEWAL_SECONDS } from "../../services";
+import { TokenCacheResponse } from "../../services/auth/token-cache.service.models";
+import { needRenewJwtToken } from "../../utilities";
+
+/**
+ * Bearer type used by supertest package.
+ */
+export const BEARER_AUTH_TYPE: { type: "bearer" } = { type: "bearer" };
```

Comment:
Just for my understanding: what does this `{ type: "bearer" } = { type: "bearer" }` mean?

Reply:
I believe it is typing the constant BEARER_AUTH_TYPE to have a property type which can hold only a value `bearer`